install-cockroachdb.md

```diff
@@ -190,21 +187,25 @@ $(document).ready(function(){
     <p>Note that at least 2GB of RAM is required to build from source. If you plan to run our test suite as well, you'll need closer to 4GB of RAM.</p>
   </li>
   <li>
-    <p>Get the CockroachDB code:</p>
-
-    <div class="highlighter-rouge"><pre class="highlight"><code data-eventcategory="mac-source-step2"><span class="gp" data-eventcategory="mac-source-step2">$ </span>go get -d github.com/cockroachdb/cockroach</code></pre></div>
+    <p>Download the <a href="https://binaries.cockroachdb.com/cockroach-latest.src.tgz">latest CockroachDB source archive</a>.</p>
   </li>
   <li>
-    <p>Build the CockroachDB binary:</p>
+    <p>Extract the sources:</p>
+    <p><div class="language-bash highlighter-rouge"><pre class="highlight"><code><span class="gp noselect shellterminal"></span>tar xfz cockroach-latest.src.tgz</code></pre></div></p>
+  </li>
+  <li><p>In the extracted directory, run <code>make</code>:</p>
 
-    <div class="highlighter-rouge"><pre class="highlight"><code data-eventcategory="mac-source-step3"><span class="gp" data-eventcategory="mac-source-step3">$ </span><span class="nb">cd</span> <span class="nv">$GOPATH</span>/src/github.com/cockroachdb/cockroach<br>$ git checkout {{site.data.strings.version}}<br><span class="gp">$ </span>make install</code></pre></div>
+    <div class="highlighter-rouge"><pre class="highlight"><code><span class="gp noselect shellterminal"></span><span class="nb">cd </span>cockroach-latest<br><span class="gp noselect shellterminal"></span>make</code></pre></div>
 
     <p>The build process can take 10+ minutes, so please be patient. The first time you run <code>make</code>, it can take even longer to download and install various dependencies.</p>
```

Comment:
One tiny nit: we no longer have to download various dependencies. We just have to build them. I might drop the second sentence entirely; you'll only be running `make` repeatedly if you're developing CockroachDB itself. So I'd recommend either of these options:

> The build process can take 10+ minutes, so please be patient.

> The build process can take 10+ minutes, so please be patient. The first time you run <code>make</code>, it can take even longer to build various dependencies.